src/gui/modals/SetupDialog.cpp

```diff
@@ -1352,4 +1357,4 @@ void SetupDialog::showRestartWarning()
 }
 
 
-} // namespace lmms::gui
\ No newline at end of file
+} // namespace lmms::gui
```

Comment:
Is this a new trend to remove those blank lines? :grin:

Reply:
I think I have Qt creator set to add a newline at EOF when saving